<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>0</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGeneral">
       <attribute name="title">
        <string>&amp;General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="optionGlobal">
          <property name="title">
           <string>Global</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="fpsL">
             <property name="text">
              <string>FPS</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundSpinBox" name="fps">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="trigger_msL">
             <property name="text">
              <string>Trigger Width</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="BoundSpinBox" name="trigger_ms">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="render_msL">
             <property name="text">
              <string>Render Width</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="BoundSpinBox" name="render_ms">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="amplificationL">
             <property name="text">
              <string>Amplification</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="BoundDoubleSpinBox" name="amplification">
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="begin_timeL">
             <property name="text">
              <string>Begin Time</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="BoundDoubleSpinBox" name="begin_time">
             <property name="maximum">
              <double>9999.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="optionAppear">
          <property name="title">
           <string>Appearance</string>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="render_resolutionL">
             <property name="text">
              <string>Resolution</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundLineEdit" name="render_resolution">
             <property name="text">
              <string>vs</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="render__bg_colorL">
             <property name="text">
              <string>Background</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="BoundColorWidget" name="render__bg_color" native="true"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="render__init_line_colorL">
             <property name="text">
              <string>Line Color</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="BoundColorWidget" name="render__init_line_color" native="true"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="render__line_widthL">
             <property name="text">
              <string>Line Width</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="BoundDoubleSpinBox" name="render__line_width">
             <property name="minimum">
              <double>0.500000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="render__grid_colorL">
             <property name="text">
              <string>Grid Color</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="OptionalColorWidget" name="render__grid_color" native="true"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="render__midline_colorL">
             <property name="text">
              <string>Midline Color</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="OptionalColorWidget" name="render__midline_color" native="true"/>
           </item>
           <item row="6" column="0">
            <widget class="BoundCheckBox" name="render__v_midline">
             <property name="text">
              <string>Vertical</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="BoundCheckBox" name="render__h_midline">
             <property name="text">
              <string>Horizontal Midline</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="optionLayout">
          <property name="title">
           <string>Layout</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="layout__orientationL">
             <property name="text">
              <string>Orientation</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundComboBox" name="layout__orientation"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="layout__ncolsL">
             <property name="text">
              <string>Columns</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="layoutDims">
             <item>
              <widget class="BoundSpinBox" name="layout__ncols">
               <property name="specialValueText">
                <string notr="true"> </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="layout__nrowsL">
               <property name="text">
                <string>Rows</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="BoundSpinBox" name="layout__nrows">
               <property name="specialValueText">
                <string notr="true"> </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStereo">
       <attribute name="title">
        <string>&amp;Stereo</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="optionStereo">
          <property name="title">
           <string>Stereo Enable</string>
          </property>
          <layout class="QFormLayout" name="formLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="trigger_stereoL">
             <property name="text">
              <string>Trigger Stereo</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundComboBox" name="trigger_stereo"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="render_stereoL">
             <property name="text">
              <string>Render Stereo</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="BoundComboBox" name="render_stereo"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="dockStereo_2">
          <property name="title">
           <string>Stereo Appearance</string>
          </property>
          <layout class="QFormLayout" name="formLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="layout__stereo_orientationL">
             <property name="text">
              <string>Stereo Orientation</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundComboBox" name="layout__stereo_orientation"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="render__stereo_grid_opacityL">
             <property name="text">
              <string>Grid Opacity</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="render__stereo_grid_opacity">
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.250000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPerf">
       <attribute name="title">
        <string>&amp;Performance</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="perfAll">
          <property name="title">
           <string>Preview and Render</string>
          </property>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="trigger_subsamplingL">
             <property name="text">
              <string>Trigger Subsampling</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundSpinBox" name="trigger_subsampling">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="render_subsamplingL">
             <property name="text">
              <string>Render Subsampling</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="BoundSpinBox" name="render_subsampling">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="perfPreview">
          <property name="title">
           <string>Preview Only</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="render_subfpsL">
             <property name="text">
              <string>Render FPS Divisor</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundSpinBox" name="render_subfps">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="render__res_divisorL">
             <property name="text">
              <string>Resolution Divisor</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="BoundDoubleSpinBox" name="render__res_divisor">
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="audioColumn">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="audioGroup">
          <property name="title">
           <string>Master Audio</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="BoundLineEdit" name="master_audio">
             <property name="text">
              <string>/</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="master_audio_browse">
             <property name="text">
              <string>&amp;Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="optionAudio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Trigger</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="trigger__edge_strengthL">
             <property name="text">
              <string>Edge Strength</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="trigger__responsivenessL">
             <property name="text">
              <string>Responsiveness</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="trigger__buffer_falloffL">
             <property name="text">
              <string>Buffer Falloff</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="BoundDoubleSpinBox" name="trigger__edge_strength">
             <property name="minimum">
              <double>-99.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="BoundDoubleSpinBox" name="trigger__responsiveness">
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="BoundDoubleSpinBox" name="trigger__buffer_falloff">
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="channelsGroup">
        <property name="title">
         <string>Oscilloscope Channels</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="channelBar">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="ShortcutButton" name="channelAdd">
             <property name="text">
              <string>&amp;Add...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ShortcutButton" name="channelDelete">
             <property name="text">
              <string>&amp;Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ShortcutButton" name="channelUp">
             <property name="text">
              <string>Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ShortcutButton" name="channelDown">
             <property name="text">
              <string>Down</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="ChannelTableView" name="channel_view"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPreview"/>
    <addaction name="actionRender"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_separate_render_dir"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionPreview"/>
   <addaction name="actionRender"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="text">
    <string>&amp;Preview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRender">
   <property name="text">
    <string>&amp;Render to Video</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_separate_render_dir">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Separate Render Folder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BoundLineEdit</class>
   <extends>QLineEdit</extends>
   <header>corrscope/gui/data_bind.h</header>
  </customwidget>
  <customwidget>
   <class>BoundSpinBox</class>
   <extends>QSpinBox</extends>
   <header>corrscope/gui/data_bind.h</header>
  </customwidget>
  <customwidget>
   <class>BoundDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>corrscope/gui/data_bind.h</header>
  </customwidget>
  <customwidget>
   <class>BoundComboBox</class>
   <extends>QComboBox</extends>
   <header>corrscope/gui/data_bind.h</header>
  </customwidget>
  <customwidget>
   <class>BoundCheckBox</class>
   <extends>QCheckBox</extends>
   <header>corrscope/gui/data_bind.h</header>
  </customwidget>
  <customwidget>
   <class>ShortcutButton</class>
   <extends>QPushButton</extends>
   <header>corrscope/gui/__init__.h</header>
  </customwidget>
  <customwidget>
   <class>ChannelTableView</class>
   <extends>QTableView</extends>
   <header>corrscope/gui/__init__.h</header>
  </customwidget>
  <customwidget>
   <class>BoundColorWidget</class>
   <extends>QWidget</extends>
   <header>corrscope/gui/data_bind.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OptionalColorWidget</class>
   <extends>QWidget</extends>
   <header>corrscope/gui/data_bind.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
